DoggyFriction/Services/DebtContainer.cs

```diff
@@ -1,29 +1,30 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using WebGrease.Css.Extensions;
 
 namespace DoggyFriction.Services
 {
     public class DebtContainer
     {
         public DebtContainerKey Key { get; }
```

Comment:
Done. Also, key now turned into DebtAggregatorKey which is private and nested inside DebtAggregator class.